### PR TITLE
[RFR] Fix Refresh in Edit and Show views

### LINKED ISSUE #1131: Refresh button does not refresh data in <ReferenceManyField>
Please do not submit support requests or "How to" questions here. For that, go to [StackOverflow](http://stackoverflow.com/questions/tagged/admin-on-rest).

**What you were expecting:**

Refresh button should refresh data in ReferenceManyField

**What happened instead:**

Data in ReferenceManyField is not refreshed (no API call) when Refresh button click

**Steps to reproduce:**

Add ReferenceManyField with reference="xyz" target="zyx", Datagrid, and TextField. Click on Refresh button.

### PR DESCRIPTION
**Pull request by fzaninotto**
The Refresh button didn't force refetch of related records (e.g. in `<ReferenceManyField>`).

Closes #1131

### CODE REVIEW

File: src/mui/detail/TabbedShowLayout.js [hunk 1]
```diff
@@ -67,16 +65,11 @@ TabbedShowLayout.propTypes = {
...
     basePath: PropTypes.string,
     translate: PropTypes.func,
+    version: PropTypes.number,
 };
 
 TabbedShowLayout.defaultProps = {
     contentContainerStyle: { borderTop: 'solid 1px #e0e0e0' },
 };
 
-const enhance = compose(
```
Comment (fzaninotto) [inline src/mui/detail/TabbedShowLayout.js]: This probably came from a bad copy/paste of the `TabbedFormLayout`. There is no `initialValues` used in the component.

**Comment by qstyler** [comment]
This refresh doesn't really work in the following case.

I have a <ReferenceManyField /> that obviously displays some related records. I have an action that removes one of the related models. I see that request to the API is fired. But the number of the records remain the same.

**Comment by fzaninotto** [comment]
@qstyler Your sentence is missing some words, please rephrase.

> I have a that obviously displays some related records.

**Comment by qstyler** [comment]
@fzaninotto  Sorry. I must have been very sleepy when I wrote this. I have created another issue for that. #1648

**Pull request merged**